pkg/apis/events/v1alpha1/storage_types.go

```diff
@@ -106,6 +106,23 @@ func (storage *Storage) GetGroupVersionKind() schema.GroupVersionKind {
 	return SchemeGroupVersion.WithKind("Storage")
 }
 
+// Methods for pubsubable interface
+
+// PubSubSpec returns the PubSubSpec portion of the Spec.
+func (s *Storage) PubSubSpec() *duckv1alpha1.PubSubSpec {
+	return &s.Spec.PubSubSpec
+}
+
+// PubSubStatus returns the PubSubStatus portion of the Status.
+func (s *Storage) PubSubStatus() *duckv1alpha1.PubSubStatus {
+	return &s.Status.PubSubStatus
+}
+
+// ConditionSet returns the apis.ConditionSet of the embedding object
+func (s *Storage) ConditionSet() *apis.ConditionSet {
+	return &StorageCondSet
```

Comment:
You might want to change this to the thing returned after you call .Manage on the condition set

Reply:
It depends on the impk of the mark methods